tests/wasmbus.rs

```diff
@@ -342,9 +342,9 @@ got: {labels:?}
 expected: {expected_labels:?}"#
             );
             ensure!(lattice_prefix == Some(TEST_PREFIX.into()));
-            ensure!(prov_rpc_host == Some("TODO".into()));
-            ensure!(rpc_host == Some("TODO".into()));
-            ensure!(uptime_human == Some("TODO".into()));
+            ensure!(prov_rpc_host == Some(ctl_nats_url.to_string()));
```

Comment:
Shouldn't these two be different?

Reply:
They could be, but aren't. We only start a single NATS server in these tests and use the same URL for all 3 connections. We should probably test that use case though: https://github.com/wasmCloud/wasmCloud/issues/520

Reply:
For clarity we could rename the variable